src/types/timestamp.rs

```diff
@@ -604,19 +604,30 @@ impl Timestamp {
     ///
     /// NOTE: This is a potentially lossy operation. A Timestamp with picoseconds would return a
     /// number of nanoseconds, losing precision. If it loses precision then truncation is preformed.
-    /// (e.g. a timestamp with fractional seconds of `0.000000000999` would be returned as `0`)
+    /// (e.g. a timestamp with fractional seconds of `0.000000000999` would return `0`)
     pub fn nanoseconds(&self) -> u32 {
         self.fractional_seconds_as_nanoseconds().unwrap_or_default()
     }
 
+    /// Returns this Timestamp's fractional seconds in microseconds
+    ///
+    /// NOTE: This is a potentially lossy operation. A Timestamp with picoseconds would return a
+    /// number of microseconds, losing precision. If it loses precision then truncation is preformed.
```

Comment:
```suggestion
    /// number of microseconds, losing precision. If it loses precision then truncation is performed.
```
This spelling mistake is present in the `nanoseconds` and `milliseconds` doc comments as well.